src/aequitas/bias.py

```diff
@@ -184,16 +324,335 @@ def get_disparity_predefined_groups(self, df, ref_groups_dict, key_columns=None,
         # after division, so if 0/0 we assume 1.0 disparity (they are the same...)
         fill_zeros = {metric: 1.000000 for metric in disparity_metrics}
         # df = df.fillna(value=fill_zeros)
+
+        # for predefined groups, use the largest of the predefined groups as
+        # ref group for score and label value
+        check_significance = \
+            [measure for measure in check_significance if measure in df.columns]
+
+        # complile dictionary of reference groups based on bias-augmented crosstab
+        full_ref_dict = {}
+        for key, val in ref_groups_dict.items():
+            full_ref_dict[key] = {'label_value': val,
+                                  'score': val}
+            for measure in check_significance:
+                full_ref_dict[key][measure] = val
+
+        # run significance method on bias-augmented crosstab based on false
+        # positives, false negatives, scores, and label values in original df
+        self.get_statistical_significance(
+            original_df, df, ref_dict=full_ref_dict, score_thresholds=None,
+            model_id=1, attr_cols=None, aplha=5e-2)
+
+        # if specified, apply T/F mask to significance columns
+        if mask_significance:
+            significance_cols = df.columns[df.columns.str.contains('_significance')]
+            truemask = df.loc[:, significance_cols] < alpha
+            falsemask = df.loc[:, significance_cols] >= alpha
+
+            df.loc[:, significance_cols] = pd.np.select(
+                [truemask, falsemask], [True, False], default=None)
+
         return df
 
 
+    @classmethod
+    def get_measure_sample(cls, original_df, attribute, measure):
```

Comment:
as currently defined, anyway, this appears to be either:

* another static method
* a module-level helper